packages/@webex/plugin-meetings/src/meeting/request.ts

```diff
@@ -177,6 +182,14 @@ export default class MeetingRequest extends StatelessWebexPlugin {
       deviceCapabilities.push(ANNOTATION.ANNOTATION_ON_SHARE_SUPPORTED);
     }
 
+    if (siHostSupported) {
```

Comment:
question: This is fine, but I thought the `deviceCapabilities` argument was added so we no longer needed to add individual flags like this?

Reply:
Thanks @Coread for pointing out the question.
It does duplicate and I have the same confusion. There are two ways to pass device capabilities in the code. I was referencing breakouts and annotation.
If `deviceCapabilities` argument is sufficient, are we considering unifying into one?

Reply:
yes please go ahead and unify into one